recipes/snp-pipeline/meta.yaml

```diff
@@ -14,7 +14,6 @@ requirements:
   build:
     - python
     - pyvcf >=0.6.7
-    - setuptools
     - psutil
     - biopython
```

Comment:
So we can remove `pyvcf`, `psutil` and `biopython` from the build requirement and only use
```yaml
  build:
    - python
    - setupools
```